library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -0,0 +1,88 @@
+
+task CalculateGeneMetrics {
+  File bam_input
+  Int estimated_required_disk = ceil(size(bam_input, "G") * 2)
+
+  command {
+    CalculateGeneMetrics -i "${bam_input}" -o gene-metrics.csv
+    gzip gene-metrics.csv
+  }
+
+  runtime {
+    docker: "quay.io/humancellatlas/secondary-analysis-sctools:master"
```

Comment:
Just adding a comment here as a reminder to change this to use a specific version of the `secondary-analysis-sctools` docker image